pkg/protocols/httpprot/request.go

```diff
@@ -406,17 +406,17 @@ func (r *builderRequest) YAMLBody() (interface{}, error) {
 
 // requestInfo stores the information of a request.
 type requestInfo struct {
-	Method   string              `json:"method" jsonschema:"omitempty"`
-	URL      string              `json:"url" jsonschema:"omitempty"`
-	Headers  map[string][]string `json:"headers" jsonschema:"omitempty"`
-	Body     string              `json:"body" jsonschema:"omitempty"`
-	FormData map[string]field    `json:"formData" jsonschema:"omitempty"`
+	Method   string              `json:"method" yaml:"method" jsonschema:"omitempty"`
```

Comment:
The megaease dedicated yaml vendor will use json tag if there is no yaml tag.

Reply:
I think making it explicit would be better because this is a "YAML in YAML" case.

Reply:
Sure, would you mind adding a line of comment about it